robocop/checkers/lengths.py

```diff
@@ -179,6 +179,22 @@
         msg="Too many test cases ({{ test_count }}/{{ max_allowed_count }})",
         severity=RuleSeverity.WARNING,
     ),
+    "0528": Rule(
+        RuleParam(name="min_calls", default=1, converter=int, desc="number of keyword calls required in a test case"),
+        rule_id="0528",
+        name="too-few-calls-in-test-case",
+        msg="Test case '{{ test_name }}' has too few keywords inside ({{ keyword_count }}/{{ min_allowed_count }})",
+        docs="""
+        Test without keywords will fail. Add more keywords or set results using Fail, Pass, Skip keywords::
+        
+            *** Test Cases ***
+            Test case
+                [Tags]    smoke
+                Skip    Test case draft
+
+        """,
+        severity=RuleSeverity.ERROR,
```

Comment:
The only concern I have is that if it's configurable, it shouldn't be an Error when someone configured the rule to 2 or more min_calls. Maybe it shouldn't be configurable and it should be named `no-keywords-in-test-case`?
We can also add one more rule that does something similar to this one you implemented but make it with Warning severity? I don't know. What are your thoughts, @bhirsz ?

Reply:
Yeaaah, if someone configure it, it would stop being an error. I think we can do something else - set the severity to Warning and incoming #614 would allow for user to set their own severity depending on value. 

Or I can later add this feature. where severity would be dynamic by default, for 0 it would throw error and 1+ warning. Basically configure severity_threshold for this rule in our code.

Reply:
I still think that maybe there should be a separate rule that throws errors when there is no keyword at all. But 2 rules may overlap if configured with the same number which would be strange...

Reply:
Both option have its pros and cons, I need to think about this. Right we can either:
 - create two separator rules, one with Error severity for 0 keyword calls
 - leave this rule, but allow to configure the severity
  - as suboption, we can by default configure error for 0 calls, and warning for 1+ calls (if someone configure it to trigger on <x )

One note that other rules like too-few-calls-in-keyword will have configurable severity depending on calls amount - so it would be more consistent to go with second option.

Reply:
Let's do the second option, like you proposed.

Reply:
Sure, it will be done after #614 is merged.